README.md

```diff
@@ -97,8 +97,8 @@ REACT_APP_REST_BASE_URL=http://localhost:8080/oscal/v1
 
 ## Contributing
 
-For the process of Contributing to the project, please review [CONTRIBUTING.md](/CONTRIBUTING.md)
-and adhere to the [Code of Conduct](/CODE_OF_CONDUCT.md).
+For the process of Contributing to the project, please review [CONTRIBUTING.md](https://github.com/EasyDynamics/.github/blob/main/CONTRIBUTING.md)
+and adhere to the [Code of Conduct](https://github.com/EasyDynamics/.github/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
```suggestion
For the process of Contributing to the project, please review [CONTRIBUTING.md][contributing]
and adhere to the [Code of Conduct][conduct].

[contributing]: https://github.com/EasyDynamics/.github/blob/main/CONTRIBUTING.md
[conduct]: https://github.com/EasyDynamics/.github/blob/main/CODE_OF_CONDUCT.md
```

For linter maybe?

Reply:
Just about to do that. Thanks!

Reply:
I just added a newline like we do in https://github.com/EasyDynamics/oscal-editor-deployment/blob/8d19c0a3cf6f87212fa70100e889f62cd2f6d8c0/README.md?plain=1#L25-#L30